libs/domain/picking/src/components/picking-lists/picking-lists.component.ts

```diff
@@ -76,6 +82,25 @@ export class PickingListsComponent extends LitElement {
     `;
   }
 
+  protected renderPickingInProgressModal(): TemplateResult {
+    return html`<oryx-modal
```

Comment:
Since you have the same modal duplicated in 2 places maybe it's good idea to move it into a component to reuse?